src/Api/Recipes/deleteSavedRecipe/index.js

```diff
@@ -1,20 +1,16 @@
+import useRemoveToken from '../../removeToken';
+import { localStorageService, recipeService } from '../../../services';
+import { axiosSavedRecipes } from '../../User';
 import { useCookies } from 'react-cookie';
```

Comment:
Please dont use axios in the name of your services, in the future you may want to change your fetching tool from aiox to something else like fetch. In that case you will have to change the name of axiosSavedRecipe in all places